components/push/storage/src/schema.sql

```diff
@@ -10,4 +12,10 @@ CREATE TABLE IF NOT EXISTS push_record (
     native_id          TEXT,
     PRIMARY KEY (uaid, channel_id)
 );
+
 CREATE UNIQUE INDEX channel_id_idx ON push_record(channel_id);
+
+CREATE TABLE IF NOT EXISTS meta_data (
```

Comment:
We probably want this to be

```
CREATE TABLE IF NOT EXISTS meta_data(
    key TEXT PRIMARY KEY,
    value TEXT NOT NULL
) WITHOUT ROWID;
```

to get the clustered index optimization.

(or even just `value NOT NULL` if non-text things will ever get stored there).

Also this is a simple enough migration that I strongly think we should just do it rather than asking our consumers to upgrade.

Reply:
Updated the `update()` method to actually update. Not super happy with it, but hard to make something generic when there's only two versions.